spec/rpc/README.md

```diff
@@ -848,6 +849,42 @@ curl -X POST https://localhost:26657 -d "{\"jsonrpc\":\"2.0\",\"id\":1,\"method\
 }
 ```
 
+### GenesisChunked
+
+Get the genesis document in a chunks to support easily transfering larger documents.
+
+#### Parameters
+
+- `chunkID`: the index number of the chunk that you wish to fetch.
```

Comment:
Personally, I think just calling it `chunk` would be better, but this is a super minor detail so I don't really mind.